app/models/profile.rb

```diff
@@ -4,6 +4,9 @@ class Profile < ApplicationRecord
   validates :data, presence: true
   validates :user_id, uniqueness: true
   validates :location, :website_url, length: { maximum: 100 }
+  validates :website_url,
+    format: { with: URI::DEFAULT_PARSER.make_regexp(%w[https http]) },
```

Comment:
Nitpick: with this formatting Rubocop complains about the alignment of the arguments to the `validates` method call. This line and the next should be aligned with `:website_url`.